docs/calendar-draft.md

```diff
@@ -245,6 +250,134 @@ Temporal.Date.from("2019-12-06").withCalendar(request.calendar).weekOfYear;
 
 The calendar IDs are less clear.  If the partial ISO calendar used ID `"iso"`, then what would the full ISO calendar use?  ID "gregory" ([why not "gregorian"?](https://github.com/tc39/ecma402/issues/212)) is misleading because there are Gregorian calendars that do not all agree on the same rules for things like weeks of the year.  One solution could be to use a nullish ID like `null` or `""` for the partial ISO calendar and `"iso"` for the full ISO calendar.  Alternatively, "iso8601", the identifier defined by CLDR as "Gregorian calendar using the ISO 8601 calendar week rules", could be the identifier for the full ISO calendar.
 
+### Methods of Construction
+
+With all four options, the calendar may be specified at the point where one of the affected Temporal types is constructed.  These methods may have different semantics when it comes to applying the default calendar.
+
+The question of default calendar only affects those methods with an ambiguous default.
+
+#### Methods With Full ISO Always Implied
+
+- Temporal.from(string)
+	- Condition: as long as a calendar ID is able to be in the string (main issue: [#293](https://github.com/tc39/proposal-temporal/issues/293)).
+- new Temporal.XYZ()
+	- The constructor is low-level and feeds the ISO data model directly.
+- HTML input
+	- The HTML5 spec only supports ISO-8601 ([reference](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#concept-date))
+
+#### Methods With Ambiguous Default
+
+- Temporal.from(fields)
+- Temporal.now
+- Temporal.Absolute.inTimeZone
+
+### Errors That Partial ISO Helps Prevent
```

Comment:
Thanks, this section is really helpful to illustrate the kinds of pitfalls that we need to protect against.

It seems these bugs all would occur if the Temporal.now functions defaulted to full ISO, because what is correct in each of these cases is for the object returned from Temporal.now to have the user's preferred calendar. Yet we cannot make the Temporal.now functions behave that way by default, because that would lead to bugs when code would run in locales where the preferred calendar isn't ISO, without having been tested by the developer. Is that an accurate statement of the problem?

I could also see this problem cropping up for Absolute.inTimeZone / TimeZone.getDateTimeFor, but I can't really see it happening for Temporal.X.from(fields) for the reason I mentioned in the previous comment.

Reply:
I'd say that's an accurate summary of the problem, yes.  Glad we're on the same page.

Yes, I realized while writing this that the problem is definitely affected by which factory method you use for constructing your Temporal.Date/Temporal.DateTime.  I think some of those methods are fine as-is, and others may need to have an explicit calendar argument.